core/src/main/scala/com/metamx/tranquility/beam/ClusteredBeam.scala

```diff
@@ -347,22 +357,31 @@ class ClusteredBeam[EventType: Timestamper, InnerBeamType <: Beam[EventType]](
         Nil
       }
     }
-    val warmingBeams = Future.collect(for (
-      latestEvent <- grouped.lastOption.map(_._2.maxBy(timestamper(_).millis)).map(timestamper).toList;
-      tbwTimestamp <- toBeWarmed(latestEvent, latestEvent + tuning.warmingPeriod) if tbwTimestamp > latestEvent
-    ) yield {
-      // Create beam asynchronously
-      beam(tbwTimestamp, now)
-    })
+    val latestEventTimestamp: Option[DateTime] = grouped.lastOption map { case (truncatedTimestamp, group) =>
```

Comment:
This changed from last event per group to last event overall. Can you explain this change a bit?